Short-Answer/Algorithms_Answers.md

```diff
@@ -19,5 +19,24 @@ This recursive algorithm calls upon itself, which means it has to resolve itself
 
```

Comment:
Great stuff Julie, like how you went with a plain English explanation _and_ some pseudocode 🔥